components/Search/SearchResultsForDocs.js

```diff
@@ -337,24 +417,102 @@ const SearchResultsForDocs = ({ apiResponse, entityType, context }) => {
               multiTagStyle={null}
               multiTagLabelStyle={null}
               isClearable={false}
+              reactSelect={{
+                styles: {
+                  menu: {
+                    width:
+                      facetValueOptsForHubs.length > 0 ? "max-content" : "100%",
+                  },
+                },
+              }}
               showCountInsteadOfLabels={true}
             />
+            <div
+              ref={publicationYearRef}
+              className="publication-year-dropdown"
+              style={{ position: "relative", zIndex: 9 }}
+              onClick={(e) =>
+                setIsPublicationYearSelectionOpen(
+                  !isPublicationYearSelectionOpen
+                )
+              }
+            >
+              <FormSelect
+                selectComponents={{
+                  Control: CustomSelectControlWithoutClickEvents,
+                }}
+                containerStyle={styles.dropdownContainer}
+                inputStyle={styles.dropdownInput}
+                options={[]}
+                value={{
+                  value: "paper_publish_year",
+                  label: selectedPublishYearRange ? (
+                    <div
+                      style={{
+                        display: "flex",
+                        columnGap: "5px",
+                        alignItems: "center",
+                      }}
+                    >
+                      {getLabelForPaperPublicationYear()}
+                    </div>
+                  ) : (
+                    <div
+                      style={{
+                        display: "flex",
+                        columnGap: "5px",
+                        alignItems: "center",
+                      }}
+                    >
+                      <span>{"Year Published"}</span>
+                    </div>
+                  ),
+                }}
+              />
+              {isPublicationYearSelectionOpen && (
+                <div
+                  onClick={(e) => e.stopPropagation()}
+                  className={css(styles.publicationYearDropdown)}
+                >
+                  <RangeSlider
+                    min={2000}
```

Comment:
Hard coded values are okay but they will be updated in subsequent PR

Reply:
Maybe add a `// TODO <comment>` for this?